var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -11,14 +11,16 @@ class Paraview(CMakePackage):
     """ParaView is an open-source, multi-platform data analysis and
     visualization application."""
 
-    homepage = 'http://www.paraview.org'
-    url      = "http://www.paraview.org/files/v5.3/ParaView-v5.3.0.tar.gz"
-    _urlfmt  = 'http://www.paraview.org/files/v{0}/ParaView-v{1}{2}.tar.gz'
+    homepage = 'https://www.paraview.org'
+    url      = "https://www.paraview.org/files/v5.6/ParaView-v5.6.2.tar.xz"
+    list_url = "https://www.paraview.org/files"
+    list_depth = 1
     git      = "https://gitlab.kitware.com/paraview/paraview.git"
 
     maintainers = ['chuckatkins', 'danlipsa']
 
     version('develop', branch='master', submodules=True)
```

Comment:
You can actually call this version `master` if you want now that #1983 has been merged.

Reply:
There is the thought that keeping the version develop will make spack more consistent even if it does not match with the name of the branch. Not sure what is better. Chuck?

Reply:
We could change it but develop has been in for a little bit now and there's still no established spack policy or practice AFAIK so I'd rather leave it as is until such a policy is in place.